docs/spec/crosschain/messages.md

```diff
@@ -192,6 +192,7 @@ message MsgVoteInbound {
 	RevertOptions revert_options = 17;
 	CallOptions call_options = 18;
 	bool is_cross_chain_call = 19;
+	string error_message = 20;
```

Comment:
I would rather have a status field using a enum InboundStatus to be consistent with voteOutbound (ReceiveStatus should be renamed to outboundStatus for this one)
Values would be for now:
- SuccessfulObservation
- InsufficientDepositirFee

Reply:
Replaced by enum `InboundStatus` https://github.com/zeta-chain/node/pull/3358/commits/3b44dc6cf18412c681ca31b4917fdcb8fccf720f